src/novo/src/client.tsx

```diff
@@ -2,27 +2,23 @@ import "desktop/lib/webpackPublicPath"
 
 import React from "react"
 import ReactDOM from "react-dom"
-import { buildClientApp } from "v2/Artsy/Router/client"
 import { getAppNovoRoutes } from "v2/Apps/getAppNovoRoutes"
 import { loadableReady } from "@loadable/component"
-import { initModalManager } from "desktop/apps/authentication/client/initModalManager"
 import { logoutEventHandler } from "desktop/lib/logoutHandler"
 import { mediator } from "lib/mediator"
 
-buildClientApp({
-  routes: getAppNovoRoutes(),
-})
+import("v2/Artsy/Router/client")
```

Comment:
this import syntax is new to me, why do we need to require it in this async way?

Reply:
Looking [here](https://github.com/artsy/force/blob/master/src/v2/Artsy/Router/client.ts#L1) it looks like we're exporting this twice -- that file is old though, we can delete it and just 

```
import { buildClientApp } from "v2/Artsy/Router/buildClientApp"
```

Since we're now only using it once (because we moved all of our new apps into the new app shell)

Reply:
@eessex  https://blog.jscrambler.com/how-to-make-your-app-faster-with-webpack-dynamic-imports/

Reply:
It's a test to see if we can reduce blocking js time and decrease tti.

https://github.com/artsy/force/pull/6741#discussion_r534561811